### ISSUE TITLE
Floating number binding goes wrong in different cultures

### ISSUE THREAD

**Issue by cotur** [abpframework/abp#3599]
I have an entity that has decimal properties. 

When I create new records int different cultures, floating point properties parsing wrong. 

![resim](https://user-images.githubusercontent.com/20099739/79210424-7c3a5280-7e4d-11ea-8df9-f5ff527187b1.png)

I send with `.(dot)` in Turkish also. But in Turkish culture floating points represented with `,(comma)` 

![resim](https://user-images.githubusercontent.com/20099739/79210961-3c279f80-7e4e-11ea-9317-e86ec5226d4e.png)

May framework support to handle it by custom modelbinder or it is required to implement individually?

**Comment by maliming** [comment]
hi @cotur  Which project can reproduce this problem?

**Comment by maliming** [comment]
By default, the model binding uses invalid culture in querystring and router to get the value, and uses the current culture to get the value in the form.

This may cause some potential binding issues. For example, we can't use the culture format (time and currency, etc.) to pass parameters in the querystring.

Solution:
1. We can use `json` to pass request parameters. Because the number format in json is fixed. But this requires more coding.

2. We can format the parameters according to the current culture in the js code and then pass it to the server. `eg: parseFloat("516.89").toLocaleString(abp.localization.currentCulture.cultureName)`

3. We can also write some custom model bindings on the server side to be compatible with some situations, but we cannot cover all.

In short, it is not easy for an application to work flawlessly in different cultures.

@cotur @hikalkan What do you guys think?

**Comment by hikalkan** [comment]
It seems it is related to UI (jquery validation) rather than the server side. Please create a new issue if you need.
